src/main/msp/msp.c

```diff
@@ -2172,6 +2176,11 @@ static mspResult_e mspProcessInCommand(uint8_t cmdMSP, sbuf_t *src)
 #else
             sbufReadU8(src);
             sbufReadU8(src);
+#endif
+#if defined(USE_ITERM_RELAX)
```

Comment:
You have included the read part into the if of the version 1.41, but this will go into version 1.42. You need your own if.

Reply:
yes, correct didnt want to make if for just one byte.